io/GltfWriter.cpp

```diff
@@ -133,8 +137,7 @@ void GltfWriter::write(const PointViewPtr v)
     TriangularMesh *mesh = v->mesh();
     if (!mesh)
     {
-        log()->get(LogLevel::Warning) << "Attempt to write point view with "
-                                      "no mesh. Skipping.\n";
+        log()->get(LogLevel::Warning) << "Attempt to write point view with no mesh. Skipping.\n";
```

Comment:
Unrelated to this PR, but are we supposed to be putting `\n`'s in log outputs?

Reply:
`log()-get()` returns an std::ostream, so you should add newlines if you want them.